src/anm/animation/animation.js

```diff
@@ -739,21 +746,30 @@ Animation.prototype.loadFonts = function(player) {
             continue;
         }
         var url = engine.checkMediaUrl(font.url),
-            woff = engine.checkMediaUrl(font.woff);
-        fontsToLoad.push(font);
-        css += '@font-face {\n' +
+            woff = engine.checkMediaUrl(font.woff), gf_name = font.gf_name;
+
+        if (!gf_name) {
+            fontsToLoad.push(font);
+            css += '@font-face {\n' +
             'font-family: "' + font.face + '";\n' +
-            'src:' +  (woff ? ' url("'+woff+'") format("woff"),\n' : '') +
-            ' url("'+url+'") format("truetype");\n' +
-            (font.style ? 'font-style: ' + font.style +';\n' : '') +
+            'src:' + (woff ? ' url("' + woff + '") format("woff"),\n' : '') +
+            ' url("' + url + '") format("truetype");\n' +
+            (font.style ? 'font-style: ' + font.style + ';\n' : '') +
             (font.weight ? 'font-weight: ' + font.weight + ';\n' : '') +
             '}\n';
+        } else {
+            url2gf += font.gf_name + "|";
+        }
+    }
+    if (url2gf != '') {
```

Comment:
it's better to use `if (url2gf.length) { ... }` to check if string is empty, or else you compare with instance of an empty string.